wagtail/wagtailcore/rich_text.py

```diff
@@ -170,3 +171,17 @@ def replace_embed_tag(m):
     html = FIND_A_TAG.sub(replace_a_tag, html)
     html = FIND_EMBED_TAG.sub(replace_embed_tag, html)
     return html
+
+
+class RichText(object):
```

Comment:
This probably needs the ``@python_2_unicode_compatible`` decorator https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.encoding.python_2_unicode_compatible